src/commands/appdistribution-distribute.ts

```diff
@@ -98,11 +103,39 @@ module.exports = new Command("appdistribution:distribute <distribution-file>")
       );
     }
 
+    if (
+      distribution.distributionFileType() === DistributionFileType.AAB &&
+      app.aabState !== AabState.ACTIVE &&
+      app.aabState !== AabState.AAB_STATE_UNAVAILABLE
+    ) {
+      switch (app.aabState) {
+        case AabState.PLAY_ACCOUNT_NOT_LINKED: {
+          throw new FirebaseError("This project is not linked to a GooglePlay account.", {
+            exit: 1,
+          });
+        }
+        case AabState.APP_NOT_PUBLISHED: {
+          throw new FirebaseError('"This app is not published in the Google Play console.', {
+            exit: 1,
+          });
+        }
+        case AabState.NO_APP_WITH_GIVEN_BUNDLE_ID_IN_PLAY_ACCOUNT: {
+          throw new FirebaseError("App with matching package name does not exist in Google Play.", {
+            exit: 1,
+          });
+        }
+        default: {
```

Comment:
If we do add `PLAY_IAS_TERMS_NOT_ACCEPTED` add a case here for it.

Reply:
+1